devito/interfaces.py

```diff
@@ -223,6 +223,36 @@ def dyz(self):
         """Symbol for the cross derivative wrt the y and z dimension"""
         return cross_derivative(self, order=int(self.space_order/2), dims=(y, z))
 
+    @property
+    def dxl(self):
```

Comment:
I'm not quite sure I understand the naming convention behind these, and the symbols `dxl` and `dyl` have the same description. Is this convention generic and understandable for general users, or would something like `dxy` and `dyx` be more appropriate? What is the general notation for these in the textbooks?